sass/public.scss

```diff
@@ -0,0 +1,235 @@
+@import '/sass/variables.scss';
+
+/**********************************/
+
+
+/*********** Nav Bar CSS **********/
+
+
+/**********************************/
+
+.navbar {
+    position: relative;
+    transition: .5s;
+    z-index: 999;
+    a {
+        text-decoration: unset;
+    }
+    &__navbar-collapse {
+        display: -ms-flexbox;
+        display: flex;
+        -ms-flex-preferred-size: auto;
+        flex-basis: auto;
+    }
+    &__navbar-brand {
+        margin: 0;
+        font-size: 45px;
+        line-height: 0px;
+        font-weight: 700;
+        letter-spacing: 2px;
+        transition: .5s;
+        img {
+            max-width: 100%;
+            max-height: 40px;
+        }
+    }
+    &__dropdown-menu {
+        margin-top: 0;
+        border: 0;
+        border-radius: 0;
+        background: $lightgrayishBlue;
+    }
+}
+
+section {
+    min-height: 300px
+}
+
+@media (min-width: 992px) {
```

Comment:
let use nest media quires with in the selector it self that will make it clearer